azure_functions_worker/dispatcher.py

```diff
@@ -244,6 +245,14 @@ async def _handle__worker_init_request(self, req):
                 result=protos.StatusResult(
                     status=protos.StatusResult.Success)))
 
+    async def _handle__worker_status_request(self, req):
```

Comment:
Is there a test we can add for this?

Reply:
Yeah, the unittest cases are added in the later iteration